ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobPropertiesSerDe.java

```diff
@@ -27,21 +27,33 @@ public SystemMetadata(long blobSize, String serviceId, String ownerId, String co
     super(blobSize, serviceId, ownerId, contentType, isPrivate, timeToLiveInSeconds);
     this.creationTimeInMs = creationTime;
   }
+
+  public SystemMetadata(long blobSize, String serviceId, String ownerId, String contentType, boolean isPrivate,
```

Comment:
nope. CreationTime is not editable in BlobProperties. guess thats why we have this intermediatery step. Since BlobProperties is also used by clients